tests/unit_tests/cli/test_scratch.py

```diff
@@ -12,14 +13,14 @@
 
 
 @pytest.fixture
-def directory_path() -> Path:  # type: ignore
+def directory_path() -> Generator[Path]:
```

Comment:
I would remove the directory_path fixture entirely in favour of the pre-existing tmp_path

Reply:
is that in the scope for this PR?

Reply:
Ack, leave it